packages/flutter_driver/lib/src/extension/_extension_web.dart

```diff
@@ -3,11 +3,13 @@
 // found in the LICENSE file.
 
 import 'dart:convert';
-import 'dart:html' as html;
-import 'dart:js';
-import 'dart:js_util' as js_util;
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
 
-/// The dart:html implementation of [registerWebServiceExtension].
+@JS('window')
+external JSObject get _window;
+
+/// The web implementation of [registerWebServiceExtension].
```

Comment:
Is there a non-web implementation of the register<ins>Web</ins>ServiceExtension? :)

Reply:
Technically yes, see `packages/flutter_driver/lib/src/extension/_extension_io.dart`. It's a no-op, but it's an implementation.